rotkehlchen/assets/utils.py

```diff
@@ -137,7 +137,12 @@ def check_if_spam_token(symbol: Optional[str]) -> bool:
         symbol.startswith('Visit https://') or
         'Please Visit' in symbol or
         (('Visit' in symbol or 'http' in symbol) and 'claim' in symbol.lower()) or
-        symbol.startswith('$') and symbol.endswith('.com')
+        (
+            symbol.startswith('$') and
+            ('.com' in symbol or '.org' in symbol or '.io' in symbol)
+        ) or
+        'at https://' in symbol or
+        ': https://' in symbol
```

Comment:
Isn't this the same, more generic and safe?

```suggestion
        ) or 'https://' in symbol
```

Reply:
I tried to be sure but we can assume that every https:// in a token is just a scam. Updated it